.gitmodules

```diff
@@ -29,7 +29,7 @@
 	url = git://github.com/mono/NUnitLite.git
 [submodule "external/buildscripts"]
 	path = external/buildscripts
-	url = git@github.com:Unity-Technologies/monobuildtools.git
+	url = https://github.com/Unity-Technologies/monobuildtools.git
```

Comment:
I wonder why this change is still, here, even after you rebased onto unity-2017-02. Let's wait for input from @joncham before we merge this.